config.go

```diff
@@ -112,6 +112,13 @@ type YNAB struct {
 	// They'd still be unapproved until approved in YNAB.
 	Cleared string `envconfig:"YNAB_CLEARED" default:"uncleared"`
 
+	// SwapFlow changes inflow to outflow and vice versa for any account with a
+	// IBAN number in the list. This is, for example, relevant for credit card
```

Comment:
Change to 'This may be relevant ...'

It is not relevant for my credit cards so it is better to indicate that it only sometimes necessary.